src/config/default_config.yaml

```diff
@@ -22,7 +22,9 @@ keys:
 
 # ethereum-package configuration
 ethereum_package:
-  additional_services:
-    - blockscout
   network_params:
     seconds_per_slot: 3
+  participants:
+    - el_type: reth
+      el_image: ghcr.io/paradigmxyz/reth
```

Comment:
We can remove this

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/94d76376293d6fc3f7ae1ac4f81c983bf6446501